### ISSUE TITLE
Option to update to pre-release versions

### ISSUE THREAD

**Issue by Riccorbypro** [seerge/g-helper#2132]
### Rules

- [X] I made myself familiar with the <a href='https://github.com/seerge/g-helper?tab=readme-ov-file'>Readme</a>, <a href='https://github.com/seerge/g-helper/wiki/FAQ'>FAQ</a> and <a href='https://github.com/seerge/g-helper/wiki/Troubleshooting'>Troubleshooting</a>.
- [X] I understand that, if insufficient information will be provided, my issue will be closed without an answer.

### Is your feature request related to a problem? Please describe

I often find myself checking the G-Helper releases page for new pre-releases. Even though there are usually no changes relevant to my laptop (2022 Zephyrus Duo), I download them anyways, so that I can report any issues I may find.  

### Describe the solution you'd like

I would like to see an option implemented (most likely in the more "power user oriented" JSON config) that allows users to opt-in to downloading prerelease versions of G-Helper.

```json
{
    ...
    "skip_aura": 1,
    "update_channel": 0, // 0 = Stable, 1 = Pre-release
    "performance_mode": 1,
    ...
}
```

### Device and Model

Asus ROG Zephyrus Duo 16 (GX650RW)

**Comment by seerge** [comment]
@Riccorbypro hello,

I actually intentionally don't want to do that. As with pre-releases it mostly collects enthousiasts (+ motivates to read release notes). And allows to check / test new things slowly.

And later I just mark one n 2-3 pre-releases as final, and push to broad public.

Next to that, there is technical limitation as app uses https://api.github.com/repos/seerge/g-helper/releases/latest API getting latest release (which points to final one). And there is no such (easy) way to get latest pre-release :)

**Comment by Riccorbypro** [comment]
That's understandable, and also the reason I suggested leaving it as an option that had to be set from the config, so "regular" users don't just go downloading them and start complaining about potential issues. I suppose those of us who do use the pre-releases will just keep watching for them, I just thought it would be a nice to have.

And yeah after posting I went and checked the GitHub APIs to see what would be needed, and found that unless I want to pull and filter a selection of all releases, I couldn't easily just grab the latest pre-release off the API, just the fully-fledged one. So yeah, that idea was kinda a bust anyways.
